capstone/backend/src/main/java/com/google/spanner/CreateSpannerTables.java

```diff
@@ -59,21 +59,25 @@ static void createDatabase(DatabaseAdminClient dbAdminClient, DatabaseId id) {
                     + ") PRIMARY KEY (UserId, ListId),"
                     + "  INTERLEAVE IN PARENT Users ON DELETE CASCADE",
                 "CREATE TABLE Stores ("
-                    + "  StoreId      INT64,"
+                    + "  StoreId      STRING(MAX),"
                     + "  StoreName    STRING(MAX),"
                     + "  Address      STRING(MAX)"
                     + ") PRIMARY KEY (StoreId)",
                 "CREATE TABLE Items ("
                     + "  ItemId            INT64,"
-                    + "  ItemNameAndBrand  STRING(MAX),"
-                    + "  ItemType          STRING(MAX)"
+                    + "  ItemName          STRING(MAX),"
+                    + "  ItemBrand         STRING(MAX),"
+                    + "  ItemType          STRING(MAX),"
+                    + "  ItemSubtype       STRING(MAX)"
                     + ") PRIMARY KEY (ItemId)",
                 "CREATE TABLE Inventories ("
-                    + "  StoreId           INT64,"
+                    + "  StoreId           STRING(MAX),"
                     + "  ItemId            INT64,"
                     + "  ItemAvailability  STRING(MAX),"
                     + "  LastUpdated       TIMESTAMP OPTIONS (allow_commit_timestamp=true),"
                     + "  Price             FLOAT64,"
+                    + "  PPU               FLOAT64,"
```

Comment:
What is PPU? Price per unit? (good to add some annotation or just name the variable PricePerUnit)

Reply:
Yeah, PPU is price per unit (so PPU might be "0.62" and unit might be "fluid oz")